### PR TITLE
fix debuginfo scoping of let-statements

### PR DESCRIPTION
**Pull request by arielb1**
r? @eddyb

### CODE REVIEW

**Comment by eddyb** [comment]
r=me after a rebase

File: src/librustc/ich/impls_mir.rs [hunk 1]
```diff
@@ -561,4 +561,4 @@ impl<'gcx> HashStable<StableHashingContext<'gcx>> for mir::ClosureOutlivesSubjec
             }
         }
     }
-}
+}
```
Comment (bjorn3) [inline src/librustc/ich/impls_mir.rs]: Missing trailing newline

**Review by bjorn3 — CHANGES_REQUESTED** [review]

**Pull request merged**